### DISCUSSION TITLE
How to Listen for System Events

### DISCUSSION THREAD

**Discussion by volkanalkilic** [nats-io/nats.net#604] [Q&A]
I'm subscribing to the leaf node connected event with a system account in the following way, but I'm not receiving any data.

```
private static async void ListenLeafNodes(NatsConnection connection) {

        var asyncEnumerable = connection.SubscribeAsync<string>("$SYS.ACCOUNT.*.LEAFNODE.CONNECT");

        await foreach (var msg in asyncEnumerable) 
            Console.WriteLine($"Received: {msg.Data}");
    }
```

**Comment by darkwatchuk** [comment]
With a system account try "$SYS.ACCOUNT.*.CONNECT" instead.  You'll get an advisory through and the client kind will be "Leafnode"
